config/examples/oauth2.yml

```
@@ -1,8 +1,11 @@
-development:
+base: &default
   github:
-    enabled: true
-    client_id: client_id
-    client_secret: client_secret
+    enabled: false
+    client_id: <%= ENV['GITHUB_CLIENT_ID'] %>
+    client_secret: <%= ENV['GITHUB_CLIENT_SECRET'] %>
```

these probably need quoting... or maybe not
```suggestion
    client_id: <%= ENV['GITHUB_CLIENT_ID'].to_json %>
    client_secret: <%= ENV['GITHUB_CLIENT_SECRET'].to_json %>
```

---

This comes from https://github.com/3scale/porta/blob/7bb79a1a802167cf5458e009dd41409f68b8cba0/config/docker/oauth2.yml#L4-L5

Why `.to_json` though?

Probably, we could just wrap in `""` (e.g. `client_id: "<%= ENV['GITHUB_CLIENT_ID']" %>`, like here: https://github.com/3scale/3scale-saas/blob/master/manifests/3scale-saas/system/overlays/pro-saas/resources/secrets/config/oauth2.yml#L5-L6

---

you may have quotes in the value, maybe not possible with a github secret though. `to_json` will escape properly for JSON, thus suitable for YAML. While implementing our own quoting/escaping is more error prone and complicated.

---

```ruby
[6] pry > 'sdsfd"fsfds'.to_json
=> "\"sdsfd\\\"fsfds\""
```